kube/services/jobs/distribute-licenses-job.yaml

```diff
@@ -96,15 +98,17 @@ spec:
             used_licenses.sort()
             print(f"Licenses currently in use: {used_licenses}")
 
-            # This is a free trial license for demo purposes only
-            # Todo: store, mount licenses secret
-            license_file = """
-            501709301583!$n1d p$53 zvqe 2sfz jzft 7aei e8yL 8ue$ j38b!snic!first line!second line!2100!
-            501709301583!$n1d p$53 zvqe 2sfz jzft 7aei e8yL 8ue$ j38b!snic!first line!second line!2100!
-            501709301583!$n1d p$53 zvqe 2sfz jzft 7aei e8yL 8ue$ j38b!snic!first line!second line!2100!
-            """.strip()
+            # Get license from kubernetes secret
+            print("Ready to read secret")
+            secret_name = os.environ['GEN3_LICENSE_SECRET_NAME']
+            secret_key ="licenseSecrets"
+            license_secrets = os.popen(
+              f"kubectl get secret {secret_name} --template={{{{.data.licenseSecrets}}}} | base64 -d"
```

Comment:
maybe add some comment in this file about how would the license secret looks like so we can have a better idea about how to create them

Reply:
Description of secrets has been added in comments. The secret key now comes from env.